CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0
+
+_Nov 12, 2021_
+
+🎉 This is the new stable release of the data grid component 🎉!
+
+If you want to migrate the DataGrid or DataGridPro from v4 to v5, take a look at the [migration guide](https://mui.com/components/data-grid/migration-v4/).
+This version is fully compatible with `@mui/material@5.X` and can be used with `@material-ui/core@4.x` with some additional steps as described [here](https://mui.com/components/data-grid/migration-v4/#using-mui-x-v5-with-mui-core-v4)
```

Comment:
What about the release blog post :) https://mui.com/blog/mui-x-v5/? I think that it should come before everything else. Similar to https://github.com/mui-org/material-ui/releases/tag/v5.0.0.

Reply:
On monday we were saying that the blogpost would come after if I remember correctly.

Reply:
> blogpost was coming after

From my perspective, the release is the blog post/published changelog. But yes, we could publish on npm today, I don't see any issue with this.

Reply:
I agree that we should have started the effort on the blog post + migration guide one week earlier to release everything at once.

Reply:
So are we going to postpone the release to after the blog post is ready? If we publish today on npm, the date in the GitHub release and the CHANGELOG won't match.

Reply:
I don't think we should postpone it.

Reply:
Actually not much is left from the blogpost - we can add the migration to it and add a `whats next` content. We can drop the 'Features highlights' section